easybuild/easyblocks/b/bamtools.py

```diff
@@ -58,10 +58,22 @@ def sanity_check_step(self):
         sharedlib_ext = get_shared_lib_ext()
 
         custom_paths = {
-            'files': ["bin/bamtools", "include/shared/bamtools_global.h", "lib/libbamtools.a",
-                      "lib/libbamtools.%s" % sharedlib_ext, "lib/libbamtools-utils.%s" % sharedlib_ext,
-                      "lib/libjsoncpp.%s" % sharedlib_ext],
-            'dirs': ["include/api", "docs"]
+            'files': [
+                "bin/bamtools",
+                "include/shared/bamtools_global.h",
+                "lib/libbamtools.a",
+                "lib/libbamtools.%s" % sharedlib_ext],
```

Comment:
I added pep8 and couldn't get it to show the error by default either. What options do you have enabled?

I just find the style fixes to be tedious and would rather know the standard and ensure the code passes a standard linter.